tests/explainers/test_tree.py

```diff
@@ -452,39 +402,29 @@ def test_catboost():
     predicted = model.predict(X)
 
     assert np.abs(shap_values.sum(1) + ex.expected_value - predicted).max() < 1e-4, \
-        "SHAP values don't sum to model output!"
-    
-    train_df, _ = amazon()
-    ix = 100
-    X_train = train_df.drop('ACTION', axis=1)[:ix]
-    y_train = train_df.ACTION[:ix]
-    X_val = train_df.drop('ACTION', axis=1)[ix:ix+20]
-    y_val = train_df.ACTION[ix:ix+20]
-    model = catboost.CatBoostClassifier(iterations=100, learning_rate=0.5, random_seed=12)
+        "SHAP values don't sum to modThisel output!"
+
+    X, y = sklearn.datasets.load_breast_cancer(return_X_y=True)
+    model = catboost.CatBoostClassifier(iterations=10, learning_rate=0.5, random_seed=12)
     model.fit(
-        X_train,
-        y_train,
-        eval_set=(X_val, y_val),        
```

Comment:
Any reason for eval_set being removed here?

Reply:
It has no purpose and takes time to evaluate.

Reply:
I guess including it does some basic testing of eval_set to make sure that nothing errors when it is set, but this could definitely be improved by adding some further testing to make sure eval_set behaves as intended.